### ISSUE TITLE
Feature: download a GIF of the animation

### ISSUE THREAD

**Issue by eimrek** [materialscloud-org/interactive-phonon-visualizer#14]
Would be great to be able to download a high-resolution GIF of the animation. Probably needs to be implemented in `weas` @superstar54.

**Comment by superstar54** [comment]
Yes, weas does not export GIF for the moment. I will add this feature.
